{{ .ProjectSnake }}/tools/yq

```diff
@@ -0,0 +1 @@
+_run_under_cwd.sh
```

Comment:
No change in `_run_under_cwd.sh` means yq will launch under multitool fine?

Reply:
yes, that's the default `case` in that script, expects the basename appears in @multitool

Reply:
And you can verify in newly added test output https://github.com/aspect-build/aspect-workflows-template/actions/runs/10967822021/job/30458181656?pr=146